bigc/resources/orders.py

```diff
@@ -25,6 +25,23 @@ def get(self, order_id: int) -> dict:
         """Get an order by its ID"""
         return self._api.v2.get(f'/orders/{order_id}')
 
+    def create(self, *, products: dict, billing_address: dict, **kwargs) -> dict:
+        """Create an order"""
+        payload = {
+            'products': products,
+            'billing_address': billing_address,
+            **kwargs,
+        }
+        return self._api.v2.post('/orders', json=payload)
+
+    def update(self, order_id: int, data: dict) -> dict:
+        """Update a specific order by its ID"""
+        return self._api.v2.put(f'/orders/{order_id}', json=data)
+
+    def delete(self, order_id: int) -> dict:
+        """Delete a specific order by its ID"""
+        return self._api.v2.delete(f'/orders/{order_id}')
```

Comment:
While the HTTP verb is `DELETE`, this actually archives an order. Also, the endpoint returns nothing according to the docs (HTTP 204).

```suggestion
    def archive(self, order_id: int) -> None:
        """Archive a specific order by its ID"""
        self._api.v2.delete(f'/orders/{order_id}')
```